run.py

```diff
@@ -95,8 +95,12 @@
                 continue
 
             # Skip build on win32
-            line = re.sub('  number: 0',
-                          '  number: 0\n  skip: true  # [win32]', line)
+            if line == '  number: 0\n':
+                line = '  number: 0\n  skip: true  # [win32]\n'
+
+            # Add sed and coreutils when make is present
+            if line == '    - {{posix}}make\n':
+                line = '    - {{posix}}make\n    - {{posix}}sed\n    - {{posix}}coreutils\n'
```

Comment:
Needs `# [win]` selector for both.

Reply:
Thanks @isuruf. I missed the `sel_src_and_win` in your PR.

I added two spaces before the selector. Alternatively I could vertically align the selector with the others. This is what is currently creates:

```
  build:
    - {{ compiler('c') }}        # [not win]
    - {{ compiler('cxx') }}      # [not win]
    - {{native}}toolchain        # [win]
    - {{posix}}filesystem        # [win]
    - {{posix}}sed               # [win]
    - {{posix}}grep              # [win]
    - {{posix}}autoconf
    - {{posix}}automake-wrapper  # [win]
    - {{posix}}automake          # [win]
    - {{posix}}pkg-config
    - {{posix}}make
    - {{posix}}sed  # [win]
    - {{posix}}coreutils  # [win]
```